.github/workflows/code_freeze_start.yml

```diff
@@ -42,10 +42,7 @@ jobs:
           page_number: 4
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Trigger AAS deploy
-        run: |
-          curl -X POST \
-          -H "Accept: application/vnd.github.v3+json" \
-          -H "Authorization: Bearer ${{ secrets.GH_TOKEN_PIERO }}" \
-          https://api.github.com/repos/DataDog/datadog-aas-extension/dispatches \
-          -d '{"event_type": "dd-trace-code-freeze", "client_payload": {"sha":"'"$GITHUB_SHA"'" } }'
\ No newline at end of file
+      - uses: ./.github/actions/deploy-aas-dev-apps
```

Comment:
Is this the right path - i.e. it uses the path from the root directory, not the current workflow directory?

Reply:
I think it is as I did the same for the code freeze action and it worked. But we'll know soon